pkg/auth/providers/common/ldap/ldap_util.go

```diff
@@ -234,8 +235,15 @@ func AttributesToPrincipal(attribs []*ldapv3.EntryAttribute, dnStr, scope, provi
 		return nil, fmt.Errorf("Failed to get attributes for %s", dnStr)
 	}
 
+	var metaName string
+	if kind == "group" || objectGUID == "" {
```

Comment:
I am not familiar with anything related to LDAP, just curious about whether a group could be represented by an GUID.

Reply:
Not sure.  I will look into that.  Could be a potential future change.